ext/posix/posix.c

```diff
@@ -1208,7 +1208,11 @@ PHP_FUNCTION(posix_pathconf)
 		Z_PARAM_LONG(name);
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (path_len == 0 || php_check_open_basedir(path)) {
+	if (path_len == 0) {
+		zend_argument_value_error(1, "Cannot be empty");
```

Comment:
The convention for `zend_argument_value_error()` is to start with a lower-case letter, so the full message reads like a sentence. Thus:
```suggestion
		zend_argument_value_error(1, "cannot be empty");
```
BTW, the test failures are likely unrelated to this PR. They should be gone when you rebase onto latest master.